src/testing/puppeteer/puppeteer-page.ts

```diff
@@ -115,6 +115,7 @@ export async function newE2EPage(opts: NewE2EPageOptions = {}): Promise<E2EPage>
 
     const failOnConsoleError = opts.failOnConsoleError === true;
     const failOnNetworkError = opts.failOnNetworkError === true;
+    const logFailingNetworkRequests = opts.logFailingNetworkRequests === true;
```

Comment:
Should we keep this as the default behavior (i.e. make it opt out)?

Reply:
ah yes, that was my intention: keep the current behavior